app/controllers/application_controller.rb

```diff
@@ -1,2 +1,26 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :authenticate_user!
+
+  def after_sign_in_path_for
+    # your path goes here
+    users_path
+  end
```

Comment:
- [ ] I got below error message while trying to sign up new user. Kindly look into and fix.

![signup-error](https://github.com/EmekaEmmanuel/Blograils/assets/26597432/ebb85471-b61e-4880-8fe0-c421d4fd38dd)